src/Gamepage/Gamepage.js

```diff
@@ -0,0 +1,40 @@
+import React, {Component} from 'react';
+import './Gamepage.css';
+
+export class Gamepage extends Component {
+  constructor(props) {
+    super(props) 
+    
+    this.state = {
+      hints: [],
+      correctGuesses: [],
+      attempts: []
+    }
+  }
+
+  render() {
+    return (
+      <div className="game-page">
+        <div className="hints">Hints
+
+        </div>
+        <div className="board">
+        <form>
+          <label>
+            Letter:
+            <input type="text" name="letter" />
+          </label>
```

Comment:
I would add a </br> at the end of the closing label tags and a space after the colons (ex: "Letter: ")